src/en/troubleshooting.md

```diff
@@ -1,15 +1,214 @@
+Title: Juju troubleshooting
 
 
 # Troubleshooting Juju
 
-This section is dedicated to the troubleshooting of Juju when things go wrong.
-Make sure you have [Juju installed](./getting-started.html#installation) and
-configured correctly.
+We know that even in the best of times things go wrong. Here we help you dive
+into what's going on and chase down those details you need when things don't
+go according to plan.
+
+## Gathering details
+
+When things don't seem to be working the first step is to pull together the
+important information that Juju is providing.
+
+
+### Filtering status
+
+`juju status` supports specifying an application to be shown. In this way,
+`juju status mysql` will only show the status information relevant to that
+application. The default output of `juju status` is a nice tabular format.
+However, sometimes there's additional details in the machine readable output.
+You can see this at any time by passing the flag `--format=yaml` to any status
+command in Juju.
+
+```bash
+
+juju status mysql --format=yaml
+model:
+  name: documentation-demo
+  controller: jaas
+  cloud: aws
+  region: eu-west-1
+  version: 2.1.2
+machines:
+  "2":
+    juju-status:
+      current: pending
+      since: 18 May 2017 13:33:58-04:00
+    dns-name: 10.96.51.68
+    ip-addresses:
+    - 10.96.51.68
+    instance-id: i-0510eb612038f780a
+    machine-status:
+      current: running
+      message: running
+      since: 18 May 2017 13:34:23-04:00
+    series: xenial
+    harGdware: arch=amd64 cores=1 cpu-power=350 mem=3840M root-disk=8192M availability-zone=eu-west-1a
+applications:
+  mysql:
+    charm: cs:mysql-57
+    series: xenial
+    os: ubuntu
+    charm-origin: jujucharms
+    charm-name: mysql
+    charm-rev: 57
+    exposed: false
+    application-status:
+      current: waiting
+      message: waiting for machine
+      since: 18 May 2017 13:33:56-04:00
+    relations:
+      cluster:
+      - mysql
+    units:
+      mysql/0:
+        workload-status:
+          current: waiting
+          message: waiting for machine
+          since: 18 May 2017 13:33:56-04:00
+        juju-status:
+          current: allocating
+          since: 18 May 2017 13:33:56-04:00
+        machine: "2"
+        public-address: 10.96.51.68
+
+```
+
+Here we're given a lot more details as to the state of things regarding the
+MySQL unit we've deployed. This includes status, timestamps, additional
+networking information, etc. When filing bugs and requesting help it's almost
+always better to use the YAML format so that everyone has additional insight
+into what's going on.
+
+
+### Beyond Juju status
+
+Juju status is a summary of what's going on. It cannot provide the full
+details all the time. Often Juju status will help provide you a hint that you
+can chase with a more specific command. One of the most important commands is
+`juju show-machine` where an application failing to deploy might be due to the
+machine never getting assigned for Juju's use.
```

Comment:
Replace paragraph with:

'The `status` command displays a status summary and is therefore the first command to use while troubleshooting. A second useful command is `juju show-machine`, which provides information on a Juju machine level.' ?